packages/@blockprotocol/hook/src/hook-service.json

```diff
@@ -56,40 +56,7 @@
         },
         "required": ["hookId"]
       },
-      "errors": [
-        {
-          "code": "NOT_IMPLEMENTED",
-          "extensions": {
```

Comment:
The core spec currently says that we just have allowed `errorCodes` defined for each message. Although it'd be good at some point to allow arbitrary additional data to be included in an error message, I don't think it's required here, since the additional info is just passing back the data of the original message (which blocks should know anyway since they have the `requestId`, or are going via a handler which abstracts it all away as a rejected promise). Therefore this felt like unnecessary extra work.